site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -1651,7 +1654,7 @@ public function ajaxAddComponent($gradeable_id) {
                 0,
                 0,
                 false,
-                false,
+                $peer,
```

Comment:
given we're not using `$peer` except here, let's delete the change on line 1636 and make this line read `$peer === 'true'` (or more preferably, `$_POST['peer'] === 'true'`)